### ISSUE TITLE
HasNativeSupport not working in SSR

### ISSUE THREAD

**Issue by ilgianfra** [jhildenbiddle/css-vars-ponyfill#16]
Hi!

I am using this lib in a project with SSR and when I run the app in the SSR I get the error that: 

> window is not defined

.
I would change the hasNativeSupport varibale in:
```
    const isBrowser = typeof window !== 'undefined';
    var hasNativeSupport = isBrowser && window && window.CSS && window.CSS.supports && window.CSS.supports("(--a: 0)");
```
What do you think? 
Thanks!

**Comment by jhildenbiddle** [comment]
Agreed, @ilgianfra!

A check for `window` should have been in place from the beginning. I'll make the update and push the update as 1.8.1.

Thanks!

**Comment by jhildenbiddle** [comment]
Fixed in 1.9.0.

Thanks again for filing the issue!
